addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss

```diff
@@ -8,16 +8,15 @@
         display: none;
     }
     .close {
+        z-index: $zindex-modal;
```

Comment:
Why is this needed?

Reply:
Otherwise, in some situations, it can disappear behind the content.

<img width="732" alt="Screenshot 2020-06-11 at 17 46 50" src="https://user-images.githubusercontent.com/7629949/84407911-c2245480-ac0b-11ea-83a2-4c5cf246fc80.png">